RecoLocalCalo/HcalRecAlgos/interface/HLTv2.h

```diff
@@ -0,0 +1,63 @@
+#ifndef HLTv2_h
+#define HLTv2_h 1
```

Comment:
Hi Stephanie,

Update of this branch is fine.
Thank you for the quick follow up.


On 3/24/15 3:31 PM, Stephanie Brandt wrote:
> In RecoLocalCalo/HcalRecAlgos/interface/HLTv2.h
> <https://github.com/cms-sw/cmssw/pull/8434#discussion_r27032470>:
> 
>> @@ -0,0 +1,63 @@
>> +#ifndef HLTv2_h
>> +#define HLTv2_h 1
> 
> I see lots of changes to make... I will begin applying suggested changes
> soon.
> Should I just push the changes to this branch again when they are done?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8434/files#r27032470>.
> 

-- 
***********************************************************************
 Vyacheslav (Slava) Krutelyov
 TAMU: Physics Dept Texas A&M MS4242, College Station, TX 77843-4242
 CERN: 42-R-027
 AIM/Skype: siava16              googleTalk: slava77@gmail.com
 (630) 291-5128  Cell (US)       +41 76 275 7116 Cell (CERN)

-----------------------------------------------------------------------